recipe/recipe.yaml

```diff
@@ -161,7 +161,7 @@ outputs:
             - fastjet-cxx
       host:
         - python
-        - ${{ pin_subpackage("fastjet-cxx", exact=True) }}
+        - fastjet-cxx
```

Comment:
The build was proceeding to the python portion with the exact versions in the non-cross compiled case, FYI.